hs_discover/views.py

```diff
@@ -84,10 +84,11 @@ def get(self, request, *args, **kwargs):
                         sqs = sqs.filter_or(owner_exact=Exact(ownertype))
             if filters.get('subject'):
                 for k, subjtype in enumerate(filters['subject']):
-                    if k == 0 or k == len(filters['subject']):
-                        sqs = sqs.filter(subject_exact=Exact(subjtype))
+                    subj = subjtype
+                    if k == 0 or k == len(subj):
+                        sqs = sqs.filter(subject_exact=Exact(subj))
                     else:
-                        sqs = sqs.filter_or(subject_exact=Exact(subjtype))
+                        sqs = sqs.filter_or(subject_exact=Exact(subj))
```

Comment:
Just curious, what is the reason to use a newly assigned variable subj instead of directly using subjtype?

Reply:
Will clean up from past experiment.